spec/features/user_can_sign_out_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+class PostsTests < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+end
+
+feature 'can sign out of an account', :devise do
+  scenario 'can sign out' do
+    user = FactoryGirl.create(:user)
+    visit '/users/sign_in'
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
+    expect(page).to have_content I18n.t 'devise.sessions.signed_in'
```

Comment:
Broke up the expectation lines and the lines that click links. I think that solves the style issue? I don't think we can group both expectations together because the second one depends on the sign out link getting clicked first.